js/modules/Methods.js

```diff
@@ -28,19 +28,25 @@ export default class Methods {
   }
```

Comment:
Kindly work on these changes:
- I'm facing with this error in chrome console:
![Screen Shot 2021-08-11 at 19 58 09](https://user-images.githubusercontent.com/22258274/129122302-ad68acb5-38e5-441d-a95b-6875bb21dc4a.png)


I can't add books.

Reply:
Hello Carlos, 

Not really sure what the issue on your side is, running this from https://mateo951.github.io/awesome-books/ should work just fine. 

Any error related to CORS it's beyond the scope of this project and our knowledge.

If you have any suggestions, they are more than welcome.

Other than that, I don't see any problem in our side.

Looking forward to your feedback.

Reply:
Hey @alexandermorales-dev, I already talked with @carlos-ssh and he will make the review with the proper link :facepunch:

Reply:
Hi team, I get the same CORS errors when trying to add a book as the previous said, please fix this and resubmit
<img width="446" alt="Screenshot 2021-08-13 at 04 16 37" src="https://user-images.githubusercontent.com/34798898/129300203-b5bf9c1c-6a00-4a56-93a7-3f7c628b765a.png">
 for re-review

Reply:
@abruzy You are testing it locally, please use the live demo link to test it out! https://mateo951.github.io/awesome-books/